corehq/apps/accounting/models.py

```diff
@@ -180,6 +180,41 @@ class PaymentMethodType(object):
     )
 
 
+class SubscriptionType(object):
+    CONTRACTED = "CONTRACTED"
+    SELF_SERVICE = "SELF_SERVICE"
+    NOT_SET = "NOT_SET"
+    CHOICES = (
+        (CONTRACTED, "Contracted"),
+        (SELF_SERVICE, "Self-service"),
+        (NOT_SET, "Not Set"),
+    )
+
+
+class ProBonoStatus(object):
+    YES = "YES"
+    NO = "NO"
+    DISCOUNTED = "DISCOUNTED"
+    NOT_SET = "NOT_SET"
+    CHOICES = (
+        (YES, "Yes"),
+        (NO, "No"),
+        (DISCOUNTED, "Discounted"),
+        (NOT_SET, "Not Set"),
+    )
+
+
+class EntryPoint(object):
+    CONTRACTED = "CONTRACTED"
+    SELF_STARTED = "SELF_STARTED"
+    NOT_SET = "NOT_SET"
+    CHOICES = (
+        (CONTRACTED, "Contracted"),
```

Comment:
much of the ops-facing UI is not translated, so I'm not going to worry about this.